ui/raidboss/data/05-shb/raid/e11s.js

```diff
@@ -614,6 +615,14 @@ export default {
           cn: '八方 -> 分摊 -> 直线 -> 击退 -> 集合',
           ko: '8산개 -> 파트너 쉐어뎀 -> 직선 장판 -> 넉백 -> 모이기',
         },
+        ttsText: {
+          en: 'Cycle of Faith: Fire',
+          de: 'Mehrfache Vergeltung: Feuer',
+          fr: 'Multi-taillade magique: Feu',
+          ja: '魔装連続剣：ファイア',
+          cn: '八方 -> 分摊 -> 直线 -> 击退 -> 集合',
```

Comment:
I think better to just drop the translation here so that find-translations will find it and somebody can translate it, otherwise nobody will ever know to change this.

Alternatively, maybe we can add some way to mark translations that need to be updated but are "good enough" for now? Maybe having find-translations look for `// FIXME` on the same line or something?

Reply:
I remember you said if tts is empty, tts doesn't play. I don't know if it was fixed, I tried to drop other TTS texts those same with text shown on screen before, but you said it will break TTS. That's why I had Chinese written same with the text on screen.

Reply:
hmmm  [this](https://github.com/quisquous/cactbot/pull/1354)?

Reply:
There must be a reason why I made commit like this, the Korean only needed TTS different from Text. [Here](https://github.com/quisquous/cactbot/pull/1014/commits/b03a28633f41af313094a26d6ff0eb9be4d09395).
If you say there's no problem, I'll remove `cn` part.

Reply:
Ah yeah, my understanding is that it works like any other trigger entry.  If you specify tts at all, then that entire `tts: {}` block overrides anything that would have come from `infoText` / `alertText`.  So, you can't just have `tts: { ja: 'string' }` because that is saying "for this trigger play something for japanese but nothing for other languages".  And, even if you have other language text in the `infoText`, if you have `tts: { en: 'something', ja: 'something else' }` but are in Korean, it will play the English version, just like any other missing text.

So, leaving out other languages mean they'll at least play the English version  until somebody translates them, which is maybe not ideal when there's at least something.

What about leaving `// FIXME` on the ones that need better translations and I'll update the translation script to find those?

Reply:
okay, I'll put `//FIXME` for later PRs. The Chinese translation is done for this PR, so I'll just use it.